### ISSUE TITLE
textsum: Format of test data

### ISSUE THREAD

**Issue by prakhar21** [tensorflow/models#692]
## Textsum
Hi, I have trained the model on the sample data already provided. How do I text the model on any arbitrary text? What should be the format of the testing data.

Thanks

**Comment by xtr33me** [comment]
Please refer to the issue https://github.com/tensorflow/models/issues/464 

You are using the included vocab file but that does not contain the words that are in the training data.  In the ticket included above, I have referenced a modified version of the vocab file that you will find a link for and if you train against that you will atleast see more than UNK.  

That said, it is very important to know that you will still not get good results unless you train against 1 million+ articles. I scraped my own 40k articles and the results still werent good, however when I brought it up to over 1 million I started seeing pretty good results. I believe the reasoning for this is due tot he fact that this is an abstractive model, which lends itself to needing a lot of data.

**Comment by wazzy** [comment]
@xtr33me 

I tried training the model with 1 million articles and 200k validation articles and 68k testing articles. I trained the model upto 320k steps now. Avg loss of training is as follows.
running_avg_loss: 2.536291
running_avg_loss: 1.118988
running_avg_loss: 4.608115
running_avg_loss: 2.832480
running_avg_loss: 1.037991
running_avg_loss: 1.361013
running_avg_loss: 3.633044
running_avg_loss: 1.186650
running_avg_loss: 2.548397
running_avg_loss: 3.287653
running_avg_loss: 3.081329
running_avg_loss: 1.456300
running_avg_loss: 0.936543
running_avg_loss: 2.312673
running_avg_loss: 1.220704
running_avg_loss: 3.213650
running_avg_loss: 5.226912
running_avg_loss: 1.159979
running_avg_loss: 0.403975
running_avg_loss: 0.805308
running_avg_loss: 1.745467
running_avg_loss: 0.306740
running_avg_loss: 1.390584
running_avg_loss: 3.714527
running_avg_loss: 1.193971
running_avg_loss: 2.119584
running_avg_loss: 3.061994
running_avg_loss: 2.613251
running_avg_loss: 1.363037
running_avg_loss: 5.849918
running_avg_loss: 1.572911
running_avg_loss: 3.726207
running_avg_loss: 2.812833
running_avg_loss: 0.895468
running_avg_loss: 3.953442

When I tried decoding it on different machine with cpu -  `bazel-bin/textsum/seq2seq_attention --mode=decode --article_key=article --abstract_key=abstract --data_path=data/data-test.bin --vocab_path=data/vocab --log_root=log_root --decode_dir=log_root/decode --beam_size=8` 

**decode15136676**
output=to DC : the New Yorker
output=<UNK> : <UNK> ?
output=Patriotism of the New Yorker
output=Is The New Yorker
output=<UNK> : <UNK> ?
output=Women 's Right : The Truth I Was
output=to DC : the New Yorker
output=to DC : the New Yorker
output=Barack in the New York Times
output=Media Is All for the New Yorker
output=<UNK> War . McCain

**ref15136676**
output=Special Rules for Credit Card Companies ?
output=Leaving Home Without It
output=So , Now , Why Have You Called Us Together ?
output=Steve Jobs , Let my Music Go
output=Leaving Home Without It
output=No Opinions ... And Proud of It !
output=Special Rules for Credit Card Companies ?
output=Special Rules for Credit Card Companies ?
output=Homeless Man Dies Saving Woman During Robbery
output=Not Like Chicken -- Fake Human Flesh Food
output=Revolution and Spirit ... ... Introduction

Do I need to still train it for some more time or do I need to make any configurations changes while doing decoding. I tried changing 

```
'max_decode_steps', 1000
'decode_batches_per_ckpt', 8
```

Please help...
